### PR TITLE
Paraplegics can choose to not have legs.

### PR DESCRIPTION
**Pull request by DATA-xPUNGED**
## About The Pull Request

This is a bounty that was made by ophaq, and claimed by me.

Gives paraplegic the option to be an amputee, and appear on station without their legs. They are still paralyzed, so no amount of mechanical legs is gonna let them walk again.

## Why It's Good For The Game

Character customization is a good thing and lets people make their little characters suffer in new ways, this one in specific *actively lowers your max HP*, which is gonna certainly be fun.

## Changelog

:cl:
qol: Paraplegics can now opt into appearing on the shift without their legs.
/:cl:

### CODE REVIEW

**Comment by Dawnseer** [comment]
This gave me the thought that a negative quirk that gives you a limp would be nice. Crutch and cane gamin

**Comment by Sadboysuss** [comment]
> This gave me the thought that a negative quirk that gives you a limp would be nice. Crutch and cane gamin

image

File: code/datums/quirks/negative_quirks/paraplegic.dm [hunk 1]
```diff
@@ -32,6 +36,18 @@
 		if(dropped_item.fingerprintslast == quirk_holder.ckey)
 			quirk_holder.put_in_hands(dropped_item)
 
+	// Finally, removes their legs if they have opted as such, deleting the shoes
+	var/amputee = GLOB.paraplegic_choice[client_source?.prefs?.read_preference(/datum/preference/choiced/paraplegic)]
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	if(amputee)
+		delete_legs(human_holder)
```
Comment (MrMelbert) [inline code/datums/quirks/negative_quirks/paraplegic.dm:43]:
```suggestion
	if(amputee)
		delete_legs(quirk_holder)
```

File: code/datums/quirks/negative_quirks/paraplegic.dm [hunk 2]
```diff
@@ -32,6 +36,18 @@
...
+
+/datum/quirk/paraplegic/proc/delete_legs(victim)
+	var/mob/living/carbon/human/human_holder = victim
```
Comment (MrMelbert) [inline code/datums/quirks/negative_quirks/paraplegic.dm:46]:
```suggestion
/datum/quirk/paraplegic/proc/delete_legs(var/mob/living/carbon/human/human_holder)
```

Comment (MrMelbert) [reply code/datums/quirks/negative_quirks/paraplegic.dm:46]: Optionally can put a type check here but I don't think it'd be necessary

**Comment by BangingDonk** [comment]
> This gave me the thought that a negative quirk that gives you a limp would be nice. Crutch and cane gamin

I would actually REALLY like this as a negative quirk, I'd love to have a middle ground between being chairbound and being perfectly mobile, because as it is, I'm just switching between the two to represent good and bad days.

**Comment by ophaq** [comment]
Nice job, can't wait to see it go live.

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**